.eleventy.js

```diff
@@ -65,35 +64,32 @@ module.exports = function (eleventyConfig) {
     }
   );
 
+  /** Converts a Date to human-readable string,
+   * e.g. 05 Oct 2021
+   */
+  const dateTimeFormatter = new Intl.DateTimeFormat('en-GB', {
+    timezone: "UTC",
+    day: "numeric",
+    month: "short",
+    year: "numeric",
+  });
+
+  /** Adds a leading zero to any number < 10 */
+  const numberFormatter = new Intl.NumberFormat('en-GB', {
+    minimumIntegerDigits: 2,
+});
```

Comment:
1️⃣ These `Intl` formatters are used in `readableDate` and `minimumTwoDigits`, respectively.